src/main/scala/sbtdatabricks/DatabricksHttp.scala

```diff
@@ -48,10 +48,19 @@ import sbtdatabricks.util.requests._
 
 /** Collection of REST calls to Databricks Cloud and related helper functions. Exposed for tests */
 class DatabricksHttp(
-    endpoint: String,
+    _endpoint: String,
     val client: HttpClient,
     outputStream: PrintStream = System.out) {
 
+  private val endpoint: String = {
+    if (_endpoint.dropRight(1).endsWith("2.")) {
```

Comment:
instead of these `dropRight`s, would it be better to use split by `/` or regex matching like `/api/([1-9.]+)` to get the API version?